src/adapters/database/AbstractSqlDatabase.h

```diff
@@ -0,0 +1,16 @@
+#ifndef SRC_ABSTRACTSQLDATABASE_H
+#define SRC_ABSTRACTSQLDATABASE_H
+
+#include <set>
+#include "../../application/DDD/Entry.h"
+
+namespace Adapters::Database
```

Comment:
Imo `Adapters` would be sufficient, as it doesn't add information to `Adapters::AbstractSqlDatabase`.

Reply:
I dont' exactly understand what you're referring to. Are you suggesting to discard the "Database" namespace inside the "Adapters" namespace? I did this because we might have a "Database::UserInterface" namespace in the future.

Reply:
Sorry I meant we will probably have a "Adapters::UserInterface" namespace. Therefore it makes sense to have a "Adapters::Database" namespace to separate between those two adapters.